src/polymer/declaration-property-handlers.ts

```diff
@@ -68,6 +68,17 @@ export function declarationPropertyHandlers(
         }
         declaration.observers.push({javascriptNode: element, expression: v});
       }
+    },
+    listeners(node: estree.Node) {
+      if (node.type !== 'ObjectExpression') {
+        return;
+      }
+      for (let prop of node.properties) {
+        if (prop.key.type !== 'Literal' || prop.value.type !== 'Literal') {
```

Comment:
Should also early exit if the `typeof` the literal values aren't strings rather than casting to string below. I think this will work and let you remove the casts:

```typescript
if (prop.key.type !== 'Literal' || typeof prop.key.value !== 'string' || prop.value.type !== 'Literal' || typeof prop.value.value !== 'string') {
  continue;
}
```

`npm run format` will handle the whitespace